packages/gasket-plugin-docusaurus/lib/configure.js

```diff
@@ -2,20 +2,23 @@ const path = require('path');
 const timing = { before: ['@gasket/plugin-docs'] };
 
 async function handler(gasket, config) {
-  const { docusaurus = {}, docs = {} } = config;
-  const { docsRoot = '.docs' } = docusaurus;
-  const { outputDir = 'docs' } = docs;
+  const { docusaurus = {} } = config;
+  const { rootDir = '.docs', docsDir = 'docs' } = docusaurus;
+
+  if (config.docs && config.docs.outputDir) {
```

Comment:
@kinetifex Moved the warning to the configure lifecycle. On line 16 the `outputDir` is set so having the check for this config property in a later lifecycle would be useless.